components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -171,10 +171,6 @@ protected AuthenticatorFlowStatus executeAutoLoginFlow(HttpServletRequest reques
         String flowType = resolveAutoLoginFlow(autoLoginCookie.getValue());
         if (AutoLoginConstant.SIGNUP.equals(flowType)) {
             alias = getSelfRegistrationAutoLoginAlias(context);
-            String domainInCookie = (String) contentJSON.get(AutoLoginConstant.DOMAIN);
-            if (StringUtils.isEmpty(domainInCookie)) {
-                throw new AuthenticationFailedException("The domain name is not available in the ALOR cookie content.");
-            }
```

Comment:
What was the reason to remove this logic?

Reply:
Domain in the autologin cookie is not used anywhere and it should not be a mandatory check